app/tests/test_ingest_hypothesis.py

```
@@ -0,0 +1,219 @@
+from __future__ import annotations
+
+import csv
+import datetime as dt
+from decimal import Decimal
+from pathlib import Path
+from typing import Dict, List, Tuple
```

Ruff (0.14.7)

`typing.Dict` is deprecated, use `dict` instead

(UP035)